### ISSUE TITLE
System.ArgumentException: Cannot dispose visual line because it is in construction!

### ISSUE THREAD

**Issue by ohadschn** [aelij/RoslynPad#136]
![image](https://user-images.githubusercontent.com/8555641/30742682-ea843648-9fa2-11e7-9c56-42e1efd25b0d.png)

> [Main Instruction]
> Unhandled Exception
> 
> [Content]
> System.ArgumentException: Cannot dispose visual line because it is in construction!
>    at ICSharpCode.AvalonEdit.Rendering.TextView.DisposeVisualLine(VisualLine visualLine)
>    at ICSharpCode.AvalonEdit.Rendering.TextView.Redraw(Int32 offset, Int32 length, DispatcherPriority redrawPriority)
>    at ICSharpCode.AvalonEdit.Highlighting.HighlightingColorizer.OnHighlightStateChanged(Int32 fromLineNumber, Int32 toLineNumber)
>    at RoslynPad.Editor.RoslynSemanticHighlighter.UpdateHighlightingSectionsNoCheck(HighlightedLine line, List`1 sections)
>    at RoslynPad.Editor.RoslynSemanticHighlighter.<>c__DisplayClass18_0.<UpdateHighlightingSections>b__0(Object o)
>    at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
>    at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
> 

Code:

```cs
public const string RFC3339DateFormat = "yyyy-MM-ddTHH:mm:sszzz";
public const string RFC3339DateFormat_UTC = "yyyy-MM-dd'T'HH:mm:ss.fffK";
DateTime? createdAfter = DateTime.UtcNow;      
createdAfter.Value.ToString(createdAfter.Value.Kind == DateTimeKind.Utc 
    ? RFC3339DateFormat_UTC 
    : RFC3339DateFormat);

createdAfter.Value.ToString(createdAfter.Value.Kind == DateTimeKind.Utc ? RFC3339DateFormat_UTC : RFC3339DateFormat).Dump();

DateTime.Now.ToString)
DateTime.UtcNow.ToString("yyyy-MM-ddTHH:mm:sszzz");
```

**Comment by SlowLogicBoy** [comment]
I'm curious if this wasn't fixed with #130 it's just not released yet :?

**Comment by aelij** [comment]
It’s not the same bug but it should also be fixed. Check again in the next version.
